src/avram/migrator/alter_table_statement.cr

```diff
@@ -166,13 +168,36 @@ class Avram::Migrator::AlterTableStatement
     ]
   end
 
-  def remove(name : Symbol)
-    dropped_rows << "  DROP #{name.to_s}"
+  {% symbol_expected_message = "%s expected a symbol like ':user', instead got: '%s'" %}
```

Comment:
This is a really cool idea 👍

Reply:
Yeah, I was really happy with that one. :blush: